pkg/manager/manager.go

```diff
@@ -379,6 +380,14 @@ func (s *FraktiManager) ContainerStatus(ctx context.Context, req *kubeapi.Contai
 	}, nil
 }
 
+func (s *FraktiManager) UpdateContainerResources(
+	ctx context.Context,
+	req *kubeapi.UpdateContainerResourcesRequest,
+) (*kubeapi.UpdateContainerResourcesResponse, error) {
+	// TODO(harry): do this for docker runtime?
+	return nil, fmt.Errorf("UpdateContainerResources is not implemented!")
```

Comment:
@feiskyer we need to run `UpdateContainerResources` for docker containers, it is required by master.